dev-lang/vala/vala-0.36.12.recipe

```diff
@@ -28,26 +32,33 @@ PROVIDES="
 	cmd:valac_${portVersion%.*}$secondaryArchSuffix
 	cmd:vapicheck_${portVersion%.*}$secondaryArchSuffix
 	cmd:vapigen_${portVersion%.*}$secondaryArchSuffix
-	lib:libvala_${portVersion%.*}$secondaryArchSuffix
-	"
-PROVIDES_common="
-	vala${secondaryArchSuffix}_common = $portVersion
+	lib:libvala_${portVersion%.*}$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	glib2${secondaryArchSuffix} >= 2.32.0
-	vala${secondaryArchSuffix}_common == $portVersion
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libdbus_1$secondaryArchSuffix
 	lib:libffi$secondaryArchSuffix
 	lib:libiconv$secondaryArchSuffix
 	lib:libintl$secondaryArchSuffix
 	"
-REQUIRES_common="
-	haiku$secondaryArchSuffix
+REPLACES="
+	vala${secondaryArchSuffix}_common
+	"
+
+PROVIDES_devel="
+	vala${secondaryArchSuffix}_devel = $portVersionCompat
+	"
```

Comment:
```diff
 PROVIDES_devel="
 	vala${secondaryArchSuffix}_devel = $portVersionCompat
+	devel:libvala_$portVers$secondaryArchSuffix = $libVersionCompat
 	"
```

Reply:
Iirc this was removed by comment?

Reply:
I don't remember why it was removed, if that was the case.
So please ignore my suggestion if you wish, because I'm not sure about this.

Reply:
This is getting too confusing ... it was you that told me to remove the line (there are no lib's in the devel package ... )

Reply:
Oops, in that case I owe you all my apologies.
I suspect that I had not noticed that the recipe was not calling `prepareInstalledDevelLib libvala-$portVers` (before the call to `	fixPkgconfig`).
And without this call we cannot have the `devel:` in `PROVIDES_devel`.
But with call, the `devel:libvala_$portVers$secondaryArchSuffix = $libVersionCompat` in `PROVIDES_devel` is OK and `haikuporter -S` is happy.

Right now I'm trying to see if we can move `$binDir` to the devel package. I think all we need in the base package is the shared lib.

Reply:
Updated, not sure yet how to move the cmd's to the devel package ...

Reply:
Cool, thanks for the update!
Yep, I haven't found yet how to use `defineDebugInfoPackage` on its files are not in the base package. So, unless someone has the answer, we can keep the `cmd:` entries in the base package.

Reply:
@fbrosson This should work:

```bash
develPrefix=$(getPackagePrefix vala${secondaryArchSuffix}_devel)
defineDebugInfoPackage vala${secondaryArchSuffix}_devel \
	"$develPrefix/$relativeBinDir/<binaries here>"

Reply:
Cool, thanks!
I'll try it.

BTW, what I have been trying to do, some time ago, was to use the `--directory` option of `defineDebugInfoPackage`, but I did not figure out how this is supposed to be used.